### ISSUE TITLE
Minor fix on start.sh script

### ISSUE THREAD

**Issue by tmigone** [balena-io-projects/google-iot#2]
It seems like IoT Core no longer accepts device ID's that start with a number, only letters.
Solved it by adding a prefix to the DEVICE_UUID on line 43.

`
gcloud iot devices create 'balena-'$RESIN_DEVICE_UUID \
         --project=$GOOGLE_IOT_PROJECT \
         --region=$GOOGLE_IOT_REGION \
         --registry=$GOOGLE_IOT_REGISTRY \
         --public-key path=rsa-cert.pem,type=rs256
`

**Comment by QuanTumli** [comment]
@tomasmigone Thanks for the input!
You also need to add the prefix to the NodeJS Script (index.js) in order to successfully publish messages for this device.

https://github.com/balena-io-projects/google-iot/blob/d7ff718e3a454a67dde1420217908cabe6777775/index.js#L11

New line should read:

```js
const deviceId = 'balena-' + process.env.RESIN_DEVICE_UUID;
```

**Comment by tmigone** [comment]
Addressed in PR #4
